packages/core/src/components/graphs/scatter-stacked.ts

```diff
@@ -66,4 +66,48 @@ export class StackedScatter extends Scatter {
 		// Add event listeners to elements drawn
 		this.addEventListeners();
 	}
+
+	getTooltipData(hoveredX, hoveredY) {
+		const domainIdentifier = this.services.cartesianScales.getDomainIdentifier();
+		const rangeIdentifier = this.services.cartesianScales.getRangeIdentifier();
+		const options = this.model.getOptions();
+		const { groupMapsTo } = options.data;
+		const percentage = Object.keys(options.axes).some(
+			(axis) => options.axes[axis].percentage
+		);
+		const stackedData = this.model.getStackedData({ percentage });
+		const tooltipData = [];
+		stackedData.forEach((groupData, groupDataIndex) => {
+			groupData.forEach((data, dataIndex) => {
+				const group = data[groupMapsTo];
+				const domainValue = data["data"]["sharedStackKey"];
```

Comment:
```suggestion
				const domainValue = datum["data"]["sharedStackKey"];
```

Reply:
Updated in https://github.com/carbon-design-system/carbon-charts/pull/764/commits/d472fea8d3d354914dd530edf463585e93af0678.